src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActorMetadata.cs

```diff
@@ -23,260 +23,89 @@ public static class KafkaConsumerActorMetadata
         /// <returns></returns>
         public static int NextNumber() => Interlocked.Increment(ref _number);
         
-        /// <summary>
-        /// Gets actor props
-        /// </summary>
         public static Props GetProps<K, V>(ConsumerSettings<K, V> settings, Decider decider) =>
-            Props.Create(() => new KafkaConsumerActor<K, V>(ActorRefs.Nobody, settings, decider, new PartitionEventHandlers.Empty(), new StatisticsHandlers.Empty())).WithDispatcher(settings.DispatcherId);
-        
-        /// <summary>
-        /// Gets actor props
-        /// </summary>
-        internal static Props GetProps<K, V>(ConsumerSettings<K, V> settings, Decider decider, IPartitionEventHandler handler, IStatisticsHandler statisticsHandler) =>
-            Props.Create(() => new KafkaConsumerActor<K, V>(ActorRefs.Nobody, settings, decider, handler, statisticsHandler)).WithDispatcher(settings.DispatcherId);
-        
-        /// <summary>
-        /// Gets actor props
-        /// </summary>
-        internal static Props GetProps<K, V>(IActorRef owner, ConsumerSettings<K, V> settings, Decider decider, IPartitionEventHandler handler, IStatisticsHandler statisticsHandler) =>
-            Props.Create(() => new KafkaConsumerActor<K, V>(owner, settings, decider, handler, statisticsHandler)).WithDispatcher(settings.DispatcherId);
+            GetProps(null, settings, decider, null);
+   
+        internal static Props GetProps<K, V>(IActorRef? owner, ConsumerSettings<K, V> settings, Decider decider, IStatisticsHandler? statisticsHandler) =>
+            Props.Create(() => new KafkaConsumerActor<K, V>(owner, settings, decider, statisticsHandler ?? StatisticsHandlers.Empty.Instance)).WithDispatcher(settings.DispatcherId);
 
 
         /// <summary>
         /// Contains <see cref="KafkaConsumerActor{K,V}"/>  message definitions.
         /// Generally this should not be used from outside the library.
         /// </summary>
         [InternalApi]
-        public class Internal
+        public static class Internal
```

Comment:
All of these messages should really have `INoSerializationVerificationNeeded` - so I added that and transformed them into `record` types as well.